public/src/components/channelManagement/campaigns/CampaignsList.tsx

```diff
@@ -56,8 +58,16 @@ const CampaignsList = ({
     return 'outlined';
   };
 
-  // When we get creation timestamps, we are thinking of allowing users to sort the list either alphabetically or by most recently created. For now, only the alphabetically sorted version is possible
-  const sortedCampaigns = [...campaigns];
+  const sortedCampaigns = campaigns.filter(c => {
```

Comment:
The `sortedCampaigns` name is quite confusing now that we're initialising it with a filter!
Perhaps the filtering could be moved to a function and then you can do something like:
`const sortedAndFilteredCampaigns = filterCampaigns().sort((a, b)...`

Reply:
I'll give it a go

Reply:
The Array `sort` function mutates the array so I've separated that out to its own function alongside the new filter function. Code now self-documents what is happening here through function/variable names

Reply:
👍